java/common/src/main/java/io/strimzi/common/TracingInitializer.java

```diff
@@ -9,14 +9,30 @@
 import io.opentracing.util.GlobalTracer;
 import io.jaegertracing.Configuration;
 
+/**
+ * Provides methods for configuring tracing
+ */
 public class TracingInitializer {
-
+    /**
+     * Initializes Jaeger Tracing by loading the configuration from the environment
+     * and registering the tracer as a global tracer
+     *
+     * @return a Jaeger Tracer object that is initialised from the environment variables
+     * using the fromEnv method
```

Comment:
maybe "using the fromEnv method" is too much detailed

Reply:
@ppatierno `using the fromEnv method` removed